pkgs/ossf/scorecard/registry.yaml

```diff
@@ -4,26 +4,27 @@ packages:
     repo_owner: ossf
     repo_name: scorecard
     description: Security Scorecards - Security health metrics for Open Source
-    asset: scorecard_{{trimV .Version}}_{{.OS}}_{{.Arch}}.tar.gz
-    files:
-      - name: scorecard
-        src: scorecard-{{.OS}}-{{.Arch}}
-    version_constraint: semver(">= 4.6.0")
-    checksum:
-      type: github_release
-      asset: scorecard_checksums.txt
-      algorithm: sha256
+    version_constraint: "false"
     version_overrides:
-      - version_constraint: semver("= 4.5.0")
+      - version_constraint: semver("< 4.4.0")
```

Comment:
Let's add test data for this.

https://github.com/ossf/scorecard/releases/tag/v4.3.1

Reply:
6312d8c4d7c46901cc4f92f1224ba72ca1039ad1